pages/index.vue

```diff
@@ -44,9 +51,9 @@ import DButton from '@/components/base/DButton';
 import DHero from '@/components/base/DHero';
 import Calendar from '@/components/calendar/Calendar';
 import { getStringDate } from '@/utils.js';
-const Vue = require('vue')
-const VueScrollTo = require('vue-scrollto')
-Vue.use(VueScrollTo)
+const Vue = require('vue');
```

Comment:
the nuxt idiomatic way to do this would be using a [plugin](https://nuxtjs.org/examples/plugins-vue) (pretty sure)

though I also wonder if using the nuxt config here could avoid the dependency on `vue-scrollto` https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-router#scrollbehavior